server/lib/wsapi/lobbies.js

```diff
@@ -143,7 +143,7 @@ export class LobbyApi {
     }),
   )
   async create(data, next) {
-    const { name, map, gameType, gameSubType } = data.get('body')
+    const { name, map, gameType, gameSubType, allowObservers } = data.get('body')
```

Comment:
So we need to make a separate change to actually make this enable-able on the client, correct?